web_crawled_data_retriever/src/web_crawling_retriever.py

```diff
@@ -176,107 +193,109 @@ def web_crawl(urls, excluded_links=None, depth = 1):
         Returns:
             tuple: A tuple containing the langchain documents (list) and the scrapped URLs (list).
         """
-        *_, web_crawling_params, extra_loaders = WebCrawlingRetrieval._get_config_info()
         if excluded_links is None:
             excluded_links = []
         excluded_links.extend(["facebook.com", "twitter.com", "instagram.com", "linkedin.com", "telagram.me", "reddit.com", "whatsapp.com", "wa.me"])
-        if depth > web_crawling_params["max_depth"]: #Max depth change with precausion number of sites grow exponentially
-            depth = web_crawling_params["max_depth"]
+        if depth > self.web_crawling_params["max_depth"]: #Max depth change with precausion number of sites grow exponentially
+            depth = self.web_crawling_params["max_depth"]
         scrapped_urls=[]
```

Comment:
Minor: "scrapped_urls" has a typo, should be "scraped_urls". Very minor issue.

Reply:
Only potential area is the hardcoded excluded links list - this could possibly be made configurable if needed for different use cases. But not a big issue but easy enough change.

Reply:
Fixed typos

Reply:
Included sites to scrape in config files